plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.test.tsx

```diff
@@ -73,9 +75,13 @@ describe('RepoUrlPicker', () => {
         >
           <SecretsContextProvider>
             <Form
+              validator={validator}
               schema={{ type: 'string' }}
               uiSchema={{ 'ui:field': 'RepoUrlPicker' }}
-              fields={{ RepoUrlPicker: RepoUrlPicker }}
+              fields={{
+                RepoUrlPicker:
+                  RepoUrlPicker as ScaffolderRJSFField<string> as ScaffolderRJSFField<string>,
```

Comment:
is this a dublication of casting to `ScaffolderRJSFField<string>` or was it intentional?:)

Could `RepoUrlPicker` implement a `ScaffolderRJSFField<string>` to avoid the multiple castings or does that not make sense?